app/scripts/modules/core/src/config/settings.ts

```diff
@@ -136,6 +136,7 @@ export interface ISpinnakerSettings {
   searchVersion: 1 | 2;
   triggerTypes: string[];
   useClassicFirewallLabels: boolean;
+  adHocInfraWrites: boolean;
```

Comment:
Do we want to make this flag name Kubernetes-specific, since it only affects the Kubernetes provider? Perhaps we could use something like `kubernetesAdHocInfraWritesEnabled`, which is perhaps a mouthful but at least descriptive?

Reply:
Changed